generators/generator-base.js

```diff
@@ -808,7 +808,28 @@ module.exports = class extends PrivateBase {
      * Generate a date to be used by Liquibase changelogs.
      */
     dateFormatForLiquibase() {
-        const now = new Date();
+        let now = new Date();
+        // Run reproducible timestamp when regenerating the project with with-entities option.
+        if (this.options.withEntities) {
+            if (this.configOptions.lastLiquibaseTimestamp) {
+                // Counter already started.
+                now = this.configOptions.lastLiquibaseTimestamp;
+            } else {
+                // Create a new counter
+                let creationTimestamp;
+                if (this.options.creationTimestamp) {
```

Comment:
I've seen this code somewhere...

Reply:
Create a function parseTimestamp on generator-base-private.js?

Reply:
or utils

Reply:
yeah maybe that'd be a good idea to put in the private thing